.appveyor.yml

```diff
@@ -5,7 +5,6 @@ environment:
   BOWER_VERSION: 1.8.8
   COMPOSER_VERSION: 5.1.0
   CONAN_VERSION: 1.18.0
-  FLUTTER_HOME: C:\flutter
   FLUTTER_VERSION: v1.12.13+hotfix.9-stable
```

Comment:
Is there a reason to keep `FLUTTER_VERSION`?

Reply:
Just for visibility. So far this version is the same that fallback version in the code, but this may change in future. When someone will want to test new version in some CI system he will not need to touch kotlin code if related env variable that control version is already in yaml.